src/ray/object_manager/plasma/connection.cc

```diff
@@ -98,11 +101,16 @@ std::shared_ptr<Client> Client::Create(PlasmaStoreMessageHandler message_handler
         }
       };
 
-  auto self =
-      std::make_shared<Client>(PrivateTag{}, ray_message_handler, std::move(socket));
-  // Let our manager process our new connection.
-  self->ProcessMessages();
-  return self;
+  ray::ConnectionErrorHandler ray_connection_error_handler =
+      [connection_error_handler](std::shared_ptr<ray::ClientConnection> client,
```

Comment:
is it possible to rename `client` here? We have a class called `Client` and another called `ClientConnection`.